### PR TITLE
[WIP] Debug mode for Docker

### PR DESCRIPTION
**Pull request by dalg24**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 version: '2'
 services:
   dtk_dev:
-    image: dalg24/dtk_stack:latest
+    image: dalg24/dtk-stack:latest
```
Comment (Rombur) [inline docker/docker-compose.yml]: oops

**Comment by aprokop** [comment]
Looks good.

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
     volumes:
       - ..:/scratch/source/trilinos/12.8.1/DataTransferKit
     environment:
       - TRILINOS_DIR=/scratch/source/trilinos/12.8.1
     container_name: dtk_dev
     command: tail -f /dev/null
+    security_opt:
+      - seccomp:unconfined
```
Comment (Rombur) [inline docker/docker-compose.yml]: you should add a comment telling why you are changing the security option

**Comment by aprokop** [comment]
Do you have anything else pending for it ([WIP])?

**Comment by dalg24** [comment]
This is up to @sslattery.  He is probably working out of this while debugging issue #98.  We can either merge this and have him open a separate PR or he can just push to that branch.

**Comment by sslattery** [comment]
lets wait to merge in case we find other issues. I have been having problems on my laptop

**Pull request merged**